web_widget_many2many_tags_multi_selection/README.rst

```diff
@@ -0,0 +1,33 @@
+Allows multiple selection on many2many_tags widget
+==================================================
+
+This module allows the user to select multiple entries through the search box in the case of the "many2many_tags" widget.
```

Comment:
The text you put on the PR (I changed it a bit):
```
In a many2many_tags widget when a lot of entries should be selected it's fastidious to select 80% of them. Then you may click on 'search more', but impossible to select several attributes at once.

This module adds a checkbox to this list so multiples entries can be selected at once.
```
would be useful in this file (more explicative than the current one I find).